src/lib/copy.py

```diff
@@ -17,7 +17,7 @@ def copy(x):
     copier = getattr(cls, "__copy__", None)
     if copier:
         return copier(x)
-    if cls in (type(None), int, float, bool, long, str, tuple, type):
+    if cls in (type(None), int, float, bool, str, tuple, type):
```

Comment:
Great question. I couldn’t think of a way to simultaneously remove long from python3 mode and keep this piece of code??

Reply:
Idea. What if I do a jseval inside the scope of this function. 

Something akin to
long = jseval(“Sk.builtins.long || Sk.builtins.int”)

Reply:
Another idea. I can add a check for isinstance since now long is an instance of int then it will be covered by that case... or __name__ == “long” some. Maybe that’s better? 🤔